Makefile

```
@@ -1,5 +1,13 @@
+all: clean lint fmt test
+
 clean:
 	rm -fr htmlcov .mypy_cache .pytest_cache .ruff_cache .coverage coverage.xml
+	hatch env remove unit
+
+install-dev:
```

```suggestion
dev:
```
We use this for python sdk